nuxt/plugin.template.js

```diff
@@ -1,29 +1,39 @@
-import Vue from 'vue'
+import Vue from 'vue';
 <% if (
   options.componentPlugins.length ||
   options.directivePlugins.length ||
   options.components.length ||
   options.directives.length
-) { %><%=
-options.componentPlugins.reduce((acc, p) => (acc += `import { ${p} } from 'bootstrap-vue/${options.dist}/components'\n` ), '') %><%=
-options.directivePlugins.reduce((acc, p) => (acc += `import { ${p} } from 'bootstrap-vue/${options.dist}/directives'\n` ), '') %><%=
-options.components.reduce((acc, c) => (acc += `import { ${c} } from 'bootstrap-vue/${options.dist}/components'\n` ), '') %><%=
-options.directives.reduce((acc, d) => (acc += `import { ${d} } from 'bootstrap-vue/${options.dist}/directives'\n` ), '') %>
+) { %>
+<% if (options.componentPlugins.length || options.components.length) { %>
+import {
+  <%= [].concat(options.componentPlugins, options.components).filter(Boolean).join(',\n  ') %>
```

Comment:
Don't we need to make sure there are no duplicated plugins?

Reply:
The plugin index de-dupes the list of component Plugins. As well, the installers will not re-install the same plugin twice.

Both components and component group plugins are available as named exports from the /xxx/components modules (similar for directives)

Reply:
This also prevents any lint warning about importing from the same module more than once.